### DISCUSSION TITLE
Traits Creator

### DISCUSSION THREAD

**Discussion by Reda-BELA** [xetra11/CK3-Workbench#133] [Feature Requests]
The Traits creator :

You can add a PNG Icon to it
Choose what stat is changed by the trait
choose what opinion is affected by the trait
Choose if the Trait Impact the Physical (like eyes color, body, ect...)
And Make that you can write a description for the trait

**Comment by xetra11** [comment]
@Reda-BELA 

Can you give me a script example of the result? That way I have a better understanding of the feature specification
